lib/student.rb

```diff
@@ -1 +1,20 @@
 
+class Student
+  attr_reader :name, :age, :scores, :grade
+  def initialize(student_info)
+    @name = student_info[:name]
+    @age = student_info[:age]
+    @scores = []
+  end
+
+  def log_score(score)
+    @scores << score
+  end
+
+  def grade 
+    scores_total = @scores.sum(0.0)
+    num_of_scores = @scores.size
+    @grade = scores_total / num_of_scores
+    @grade
```

Comment:
We don't need an instance variable here since it's not an attribute we're defining at instantiation. In fact you can just return 17 as the last line and this should still function correctly.
```suggestion
    scores_total / num_of_scores
```